src/openzaak/management/commands/setup_configuration.py

```diff
@@ -80,6 +84,22 @@ def handle(self, *args, **options):
             notif_config.save()
 
             # Step 2
+            # This service should be created in migration
+            #   0002_move_config_to_service_model
+            service = Service.objects.get(label="Notificaties API")
```

Comment:
I'd rather not rely on a hardcoded name, but on the service type:

```suggestion
            service = Service.objects.filter(api_type=APITypes.nrc).first()
            assert service is not None, "Service should have been created by migrations"
```